.github/workflows/cherryPick.yml

```diff
@@ -71,7 +71,8 @@ jobs:
       - name: Cherry-pick the version-bump to staging
         run: |
           # Find the commit for the version-bump
-          git fetch origin main
+          # This command fetches commits in the last hour on the main branch
+          git fetch origin main --no-tags --shallow-since="$(( $(date -%s) - 3600 ))"
```

Comment:
I could be wrong here, but isn't this ensuring that we only pull a limited amount of commits? Essentially, it's dropping the entire history of commits that we are currently pulling (which could be contributing to the current HTTP Error)

Reply:
Though yeah, I see your concern. Maybe we could limit it to 24 hours?

Reply:
Great catch @luacmartins. You're right and that wasn't the intention at all. I've updated this now...

- We fetch commits on the main branch in the last hour. This should always work for the version bump, which will have just happened (via the callable workflow on line 30 of this file)
- Then if the merge commit of the PR we are trying to CP is not found, fetch it (w/ `--depth=1`, because we only need the one commit).

Again, great catch!